### DISCUSSION TITLE
[ARCHIVED] Museum Project - Wireframe prototype

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#11] [Announcements]
Post a link to your Figma prototype

It should include rough versions of:

- the different menus (start screen, potential settings, about pages, etc.)
- representation of the collections (either in AR, in 2D, as 3D objects)
- unique user features (the ones that would differentiate your app from other apps)

Come prepared to discuss it in class.

**Comment by yylinj** [comment]
HI, please find the drive link attached. 
https://docs.google.com/document/d/1ATmMYKcqtqaRfQtUfEX3OVsK0a806K-xAn4KVM_bFKQ/edit?usp=sharing

**Comment by vb1424** [comment]
https://docs.google.com/document/d/1xuiuHTCMGHznJUYdpAgF-4cqWV-JHieYKpVSocMWtNk/edit#heading=h.fgksoju57dp

**Comment by zanderzel** [comment]
https://docs.google.com/document/d/1d2auXazB5MDKSATYApaQL6qWLK3cKErbGZmv6pkftuk/edit?usp=sharing

**Comment by palomabrites** [comment]
https://docs.google.com/document/d/1kvBM95gbx_PNeCM2shTvPjGNQNQREerHUiszmfNX144/edit?usp=sharing

**Comment by Mg5970** [comment]
https://docs.google.com/document/d/1PHJ-_pBW904vbux_AhXFMYPhG88Y1hDi9QoQud_TBnA/edit?usp=sharing

**Comment by aidanmassie** [comment]
https://docs.google.com/document/d/12WfWpBuC_yWg3-fWnyosVtWFUN10SwRL6Esd267XXVM/edit?usp=sharing

**Comment by Lkochar19** [comment]
https://docs.google.com/document/d/1BNEc61OTpZHrrO3n_a8uCVCA5BFoZANaitEIdEHl0WY/edit?usp=sharing

**Comment by chloeumlaut** [comment]
https://docs.google.com/document/d/1Be-Lmh6gfbUdNE_LHjLe9HaW_g3LhMq9ZDNiHdfNHD0/edit?usp=sharing

**Reply by brianajpierre** [reply]
WireFrame Sketch - https://drive.google.com/file/d/1Y0I6LA1tgoGzCtoFEJjVlJ1SqcTA6Xkz/view?usp=sharing

**Comment by izzyrieken** [comment]
David and Isabelle - Humboldt Forum Architecture

Google Doc: https://docs.google.com/document/d/18lWz6lMaN3iJcUS2CSfCPH_e_oEFRmwFjes2GhMqmF0/edit

**Comment by olivestew222** [comment]
Alaya, Hubert, and Olive - Humboldt Forum Adventure Game

Doc: https://docs.google.com/document/d/13I-Ih61xrzHnuQLOxOqX5SyOetWaph4mGeWzmR0tHzQ/edit?usp=sharing

**Comment by IanPort** [comment]
Dove, Maria, and Ian - Future Walk 1

https://docs.google.com/document/d/1khtnGRMoMeiqCOFX4Tw4srZyPN3XKzerw9omftiLvB0/edit?usp=sharing

**Comment by kenparks12** [comment]
Kendall, Vasant, Kyle - Future Walk

Doc: https://docs.google.com/document/d/16fQ6P1IMamC095ptxjJUbHrH3KFam7KRJj5TMK5fgpM/edit?usp=sharing

**Comment by aacarcrash** [comment]
Working with @hannah-y-zhao

[Figma Link](https://www.figma.com/file/fD5Sy5jeJgugTvBHIkr9Oh/Komposita?node-id=4%3A2&t=fOhJQLfg9QbNCEc8-1)


The app works by prompting the user to move closer to the pillars. After that, 2 drop downs appear. The user can select a root word from each and hit the fusion to make their own fuesd german compound sculptures. The label on the pedestal will give an english translation as well, hence making this a language learning experience too.

**Reply by hannah-y-zhao** [reply]
Thinking of using separate materials and models; first word selected determines the material, second word determines model.

**Comment by Yupu-Chen** [comment]
Working with Samuel, we are creating an AR application that allows users to curate their own exhibitions. In other words, we are making a virtual version of the DIY Archive. More details about the application can be seen here:

https://www.figma.com/file/nUItkQYC7b0m3ze4JFzamg/Augmenting-Gallery?node-id=0%3A1&t=unHmt4cpekHfoydz-1

**Comment by cz2109** [comment]
Working with Emerson, we are creating an AR application that involves a curated music collection. Still deciding on how to source the collection itself, the prototype uses Rolling Stone magazine as an example, but this is still up for discussion. Using the prototype feature, you can navigate via clicking on buttons and album covers. In the actual app, the latter will involve the user moving closer to the album along the timeline to trigger audio/information.

https://www.figma.com/file/lSUOCIFLjzjtEpaxfhXv2Q/Enter-the-World-of-Music?node-id=0%3A1&t=jDpxiJaKpSWohxHc-1

**Comment by mariamabdelrazekk** [comment]
Me an Dania are creating an AR application that provides more information about the objects in the Crochet museum. Given that the objects displayed do not have historical value, the information we will provide on the objects will be about the technicalities, focusing on the stitched used and trying to help the audience understand how the pieces were created. The first page in our application gives the visitors an option of stating whether they are physically at the museum or not, if they are, they can scan QR codes near the objects at the museum to retain the information. If they are not physically at the museum, they will get to chose from a bulk of the objects displayed, and information will be displayed about this category of objects, as well as a 3D rendering of the object that enables them to view it in whatever their surrounding is. 

https://www.figma.com/file/rA8LfJyrv9DPerC5mY66uS/Final-Project-Prototype?node-id=0-1&t=VyjKetuKLdoGmGSP-0
